packages/fx-core/src/plugins/resource/utils4v2.ts

```diff
@@ -404,6 +404,9 @@ export async function configureLocalResourceAdapter(
   if (isConfigUnifyEnabled() && envInfo) {
     pluginContext.envInfo.state = objectToMap(envInfo!.state);
   }
+  if (envInfo?.config.isLocalDebug) {
```

Comment:
both provisionLocal "& configureLocal should add this value.

Reply:
I will add ```provisionLocal``` later. Thanks